src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -317,7 +344,8 @@ namespace
 
             // TODO: Encode no alias attribute as part of the function signature conversion or as a
             // separate rewrite pattern. Retrieve new function after signature conversion.
-            insertNoAliasArgAttrs();
+            // TODO: To be enabled in follow-up commit.
```

Comment:
Is disabling this needed for the MLIR update ? If not, then we should keep it.

Reply:
Yes, it's needed due to the new calling convention. Otherwise tests will fail since we are adding `noalias` to arguments that are not pointers.

Reply:
I'm enabling this in a follow-up patch for the bare ptr calling convention.